src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StandaloneFormula.kt

```diff
@@ -227,14 +231,14 @@ class StandaloneFormula private constructor(
         val setupDatabase = executor.submitWithLogContext("database") {
             databaseSsh.newConnection().use {
                 databaseComputer.setUp(it)
-                logger.info("Setting up database...")
+                logger.info("Setting up database with ip ${databaseSshIp}...")
```

Comment:
Why log IP in this specific case? Should all the other provisioning logs include IPs?

Reply:
it helped a lot during analyzing problems allowing going straight to DB host.
I've also raised https://ecosystem.atlassian.net/browse/JPERF-761 - I think it should be much easier to ssh to any provisioned host

Reply:
That feature is already done, see the Jira comment.

Reply:
I've checked it and indeed it is. I've removed the ip from logs